core/capabilities/triggers/logevent/service.go

```diff
@@ -0,0 +1,143 @@
+package logevent
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+)
+
+const ID = "log-event-trigger-%s-%d@1.0.0"
+
+const defaultSendChannelBufferSize = 1000
+
+// Log Event Trigger Capability Input
+type Input struct {
+}
+
+// Log Event Trigger Capabilities Manager
+// Manages different log event triggers using an underlying triggerStore
+type TriggerService struct {
+	capabilities.CapabilityInfo
+	capabilities.Validator[RequestConfig, Input, capabilities.TriggerResponse]
+	lggr           logger.Logger
+	triggers       CapabilitiesStore[logEventTrigger, capabilities.TriggerResponse]
+	relayer        core.Relayer
+	logEventConfig Config
+}
+
+// Common capability level config across all workflows
+type Config struct {
+	ChainID        string `json:"chainId"`
+	Network        string `json:"network"`
+	LookbackBlocks uint64 `json:"lookbakBlocks"`
+	PollPeriod     uint32 `json:"pollPeriod"`
+}
+
+func (config Config) Version(capabilityVersion string) string {
+	return fmt.Sprintf(capabilityVersion, config.Network, config.ChainID)
+}
+
+type Params struct {
+	Logger         logger.Logger
+	Relayer        core.Relayer
+	LogEventConfig Config
+}
+
+var _ capabilities.TriggerCapability = (*TriggerService)(nil)
+var _ services.Service = &TriggerService{}
+
+// Creates a new Cron Trigger Service.
+// Scheduling will commence on calling .Start()
+func NewTriggerService(p Params) *TriggerService {
+	l := logger.Named(p.Logger, "LogEventTriggerCapabilityService")
+
+	logEventStore := NewCapabilitiesStore[logEventTrigger, capabilities.TriggerResponse]()
+
+	s := &TriggerService{
+		lggr:           l,
+		triggers:       logEventStore,
+		relayer:        p.Relayer,
+		logEventConfig: p.LogEventConfig,
+	}
+	s.CapabilityInfo, _ = s.Info(context.Background())
```

Comment:
We should handle the error here; even if it doesn't return an error today, it may in the future, and we shouldn't require the person making that future change to go and check all of the places this is called to ensure they're handling errors correctly.

Reply:
And pass a real context

Reply:
Sure, Added error check here